### PR TITLE
[MRG] Update: Changed and added Meta Description

### PR DESCRIPTION
**Pull request by TillLord**

### CODE REVIEW

**Comment by betatim** [comment]
Did you discuss these changes with @kirastina? (Showing Kira how Github works)

**Comment by Kirastina** [comment]
He doesn't need to :)

File: site/content/english/expert-suisse.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Skribble – Legally binding electronic signatures with Skribble
-description: Easily handle digital signature processes – legally binding in accordance with Swiss & EU law.
+title: Preferential price for Expert-Suisse members
+description: Legally binding electronic signatures with Skribble. Easily handle digital signature processes – legally binding in accordance with Swiss & EU law.
```
Comment (betatim) [inline site/content/english/expert-suisse.md]:
```suggestion
description: Legal-ly binding electronic signatures with Skribble. Easily handle digital signature processes – legally binding in accordance with Swiss & EU law.
```

**Comment by TillLord** [comment]
Ist es, weil ich ihr Git-Hub zeigen soll, oder weil wir das 4 Augen-Prinzip möchten?

Von: Tim Head <notifications@github.com>
Antworten an: BlockSigner/cove <reply@reply.github.com>
Datum: Montag, 30. September 2019 um 15:06
An: BlockSigner/cove <cove@noreply.github.com>
Cc: Till <till@skribble.com>, Author <author@noreply.github.com>
Betreff: Re: [BlockSigner/cove] [MRG] Update: Changed and added Meta Description (#184)


Did you discuss these changes with @Kirastina<https://github.com/Kirastina>? (Showing Kira how Github works)

**Comment by betatim** [comment]
We were experimenting with using GitHub.

**Comment by betatim** [comment]
I'll merge this now.

**Pull request merged**